modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/conversion/swagger/v2_0/SwaggerUtils.java

```diff
@@ -57,6 +64,23 @@
     private SwaggerUtils() {
     }
 
+    public static Object getDefaultValue(Property swaggerProperty) {
+        if (swaggerProperty instanceof IntegerProperty) {
+            return ((IntegerProperty) swaggerProperty).getDefault();
+        } else if (swaggerProperty instanceof LongProperty) {
+            return ((LongProperty) swaggerProperty).getDefault();
+        } else if (swaggerProperty instanceof FloatProperty) {
+            return ((FloatProperty) swaggerProperty).getDefault();
+        } else if (swaggerProperty instanceof DoubleProperty) {
+            return ((DoubleProperty) swaggerProperty).getDefault();
+        } else if (swaggerProperty instanceof UUIDProperty) {
+            return ((UUIDProperty) swaggerProperty).getDefault();
+        } else if (swaggerProperty instanceof StringProperty) {
+            return ((StringProperty) swaggerProperty).getDefault();
+        }
+        return null;
```

Comment:
A log (level: warning) would be nice here to prevent us from forgetting to add new types when we update the Swagger version.